deploy/cro-aws-config.yml

```diff
@@ -0,0 +1,9 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cloud-resources-aws-strategies
```

Comment:
Could we make this name `cloud-resource-config` as this is the default used for this config map when it's installed by rhmi - https://github.com/integr8ly/integreatly-operator/blob/master/pkg/controller/installation/installation_controller.go#L48

Reply:
`cloud-resource-config` is a different [config map](https://github.com/integr8ly/integreatly-operator/blob/b161ee20f273ae59e31f22301f095fcf6ebe2ccf/pkg/controller/installation/bootstrapReconciler.go#L116-L117), it is for defining tiers, where as this `cloud-resource-aws-strategies` is for defining the strats for the resources.

Reply:
Yep, to clarify `cloud-resource-config` defines which provider will be invoked e.g. `aws` or `openshift`(in the future `gcp`)

`cloud-resource-aws-strategies` is specific config for the AWS provider e.g. what an RDS instance should look like, what a VPC should look like etc.

Reply:
ah right. yeah saw the change in this pr but didn't read the data so assumed they were the same thing. nvm. 👍